src/js/languages/default.json

```diff
@@ -8,6 +8,11 @@
     "dropPrompt": "Drop file here or",
     "dropPromptMultiple": "Drop files here or",
     "files": "files",
+    "maxSizeSingle": "The file is too large. Select a file no larger than {maxSize} bytes.",
```

Comment:
Maybe an enhancement for later but any way to convert bytes into MB, GB, etc. where applicable since a large number of bytes may be unfamiliar to users especially since file browsers convert the values

Reply:
Absolutely agree. I know we discussed an enhancement being the ability to specify our `maxSize` in some sort of shape that would look like:

```js
const maxSize = {
  unit: 'MB',
  size: 5
} // ==> 5MB
```

But I will try and see if I can implement a function to parse the current bytes implementation into a more readable format inside the message for now.

Reply:
I would recommend just always specifying the maxSize in bytes and let the code decide how to best format it for display.

Reply:
I implemented a function to parse the bytes into a readable format and adjusted the messaging accordingly.